src/main/java/org/jabref/model/entry/Date.java

```diff
@@ -21,6 +21,18 @@ public class Date {
     static {
         List<String> formatStrings = Arrays.asList(
                 "uuuu-MM-dd'T'HH:mm:ss[xxx][xx][X]",    // covers 2018-10-03T07:24:14+03:00
+                /*
+                 start: fix 8754
+                 */
```

Comment:
Please remove that comment, unnecessary. Please also add some tests for the new format